scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,93 @@
 
 # hardcode the upgrade you want to do here.
 def get_upgrade_set():
-    state_exp_settings = StateExpirationSettings(max_entry_expiration=Uint32(6312000),
-                                                 min_temp_entry_expiration=Uint32(
-                                                     16),
-                                                 min_persistent_entry_expiration=Uint32(
-                                                     86400),
-                                                 auto_bump_ledgers=Uint32(0),
-                                                 persistent_rent_rate_denominator=Int64(
-                                                     0),
-                                                 temp_rent_rate_denominator=Int64(
-                                                     0),
-                                                 max_entries_to_expire=Uint32(
-                                                     0),
-                                                 bucket_list_size_window_sample_size=Uint32(
-                                                     0),
-                                                 eviction_scan_size=Uint64(0))
-
-    entry = ConfigSettingEntry.from_config_setting_state_expiration(
-        state_exp_settings)
-    return ConfigUpgradeSet([entry])
+
+    max_data_entry_size = 64 * 1024
+
+    max_contract_size = Uint32(max_data_entry_size) 
+
+    contract_size_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES,
+        contract_max_size_bytes = max_contract_size)   
+        
+    compute_settings = ConfigSettingContractComputeV0(ledger_max_instructions=Int64(1_000_000_000),
```

Comment:
The ledger-wide limits for phase 1 are supposed to be equal to the tx limits (here in particular and in all the other occurences).